java/flight/flight-sql/src/main/java/org/apache/arrow/flight/sql/FlightSqlClient.java

```diff
@@ -346,7 +345,7 @@ public FlightInfo getExportedKeys(final TableRef tableRef, final CallOption... o
     }
 
     Objects.requireNonNull(tableRef.getTable());
-    builder.setTable(tableRef.getTable()).build();
```

Comment:
It seems the right fix should be to just remove `.build`? Or else, inline it below or pull out some code? e.g. 

`Any command = Any.pack(builder.setTable(...).build());`

Reply:
If it is deleted `builder.setTable(tableRef.getTable()).build();`, errors appear like these:
````
[ERROR] Failures: 
[ERROR]   TestFlightSql.testGetCommandExportedKeys:589 expected:<1> but was:<0>
[ERROR]   TestFlightSql.testGetCommandImportedKeys:619 expected:<1> but was:<0>
[ERROR] Errors: 
[ERROR]   TestFlightSql.testGetPrimaryKey:505 » IndexOutOfBounds Index 0 out of bounds f...
[INFO] 
[ERROR] Tests run: 47, Failures: 2, Errors: 1, Skipped: 0

````

Reply:
The warning is only saying that you're ignoring the reuslt of `build()`. So you can presumably remove the `build`, but keep the rest of the line.

Reply:
If I change to `Any command = Any.pack(builder.setTable(tableRef.getTable()).build());` it is working but I don't know what Any.pack is doing behind the scenes or if it is ok to use that instead of `initialize`method?

Reply:
Can you clarify? `isInitialized` doesn't do anything, as the method documentation says.

Reply:
Decided to move on your previous comment and remove the `build` but keep the rest of the line.